docs/userguide.md

```diff
@@ -69,6 +69,18 @@ factoryResource.use { factory =>
 ## Extensions
 Extensions are packages that monitor some functionality for you - without you having to do anything.
 
+### Http4s
```

Comment:
please add this to table of contents above

Reply:
done. it's a shame it can't be generated automatically though:(